scalalib/src/mill/scalalib/GenIdeaImpl.scala

```diff
@@ -58,6 +58,18 @@ object GenIdeaImpl {
     }.getOrElse(None)
   }
 
+  private val millModules = Seq(
```

Comment:
Is mill exporting IDEA projects for these modules in case the user wants to modify them? Or what is going on here? :smile:

Reply:
It's exporting these as idea "libraries" so that the user can `jump to definition` from their build file ^^

Reply:
Aha, thanks! :smile: